libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -889,7 +892,13 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
         cmd.content.location.alt = packet.z * 100.0f;       // convert packet's alt (m) to cmd alt (cm)
 
         switch (packet.frame) {
-
+        case MAV_FRAME_LOCAL_NED:
```

Comment:
Can this be wrapped in a BUILD_TYPE preprocessor check? On vehicles other then copter I think we should still always be reporting that it is unsupported, so that GCS's don't try and put MAV_FRAME_LOCAL_NED waypoints on a plane.

Reply:
only for plane then, rover will definitivly use this. I bet that sub too

Reply:
OK. It only get built in APM_BUILD_ArduCopter